packages/integration-tests/projects/suite-web/tests/onboarding/hologram.test.ts

```diff
@@ -1,21 +1,27 @@
 // @group:onboarding
 // @retry=2
 
-describe('Onboarding - hologram', () => {
+// todo: is there any hologram in der neue onboarding?
```

Comment:
yes, sir. but only for device without fw installed, and to make things more complicated it is inside a tooltip.
<img width="608" alt="Screenshot 2021-04-28 at 21 35 13" src="https://user-images.githubusercontent.com/6961901/116462334-a056af00-a869-11eb-8289-40611199a433.png">